main/res/xml/preferences.xml

```diff
@@ -921,7 +921,7 @@
         <cgeo.geocaching.settings.SeekbarPreference
             android:key="@string/pref_list_initial_load_limit"
             app:min="0"
-            app:max="5000"
+            app:max="50000"
             android:defaultValue="1000" />
```

Comment:
Interesting though: In the settings the default is already set to 0.

https://github.com/cgeo/cgeo/blob/51162ae0f986506b6806d4abc068b5ed3d248e9a/main/src/cgeo/geocaching/settings/Settings.java#L1919

However, a number resource should be used for the default value, so that always the same value is used at both places...

Reply:
Will add that